.happy/terraform/envs/rdev/main.tf

```diff
@@ -0,0 +1,17 @@
+module stack {
+  source              = "./modules/ecs-stack"
+  aws_account_id      = var.aws_account_id
+  aws_role            = var.aws_role
+  happymeta_          = var.happymeta_
```

Comment:
why does happymeta_ have the trailing underscore? (just for my own understanding)

Reply:
This is a private value - don't touch!